packages/app/app.py

```diff
@@ -1,11 +1,46 @@
 from flask import Flask, render_template, request, redirect, url_for
 import sqlite3 as sql
 from sqlite3 import Error
+import requests
+import json
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
 
 # Configure application
 app = Flask(__name__)
 
-# Connect to db function
+def yelp_lookup(city, state):
+    #city = "New York"
+    #state = "NY"
+
+    # Contact API
+    #try:
+    API_KEY = os.getenv("API_KEY")
+    headers = {"Authorization": "Bearer {0}".format(API_KEY)}
+    url = ("https://api.yelp.com/v3/businesses/search?term=donut&location={city},{state}")
```

Comment:
Does this work? I've only seen this style of setup with f strings, though in that case I think I'd expect to see an `f` prefixing the double quotes.

Ref: https://realpython.com/python-f-strings/

Reply:
You are correct. This does not work. I tried this format as an f string but it kept throwing errors. It works if I'm not putting params directly into the url. If you were notified to review, apologies, that was a mistake. Perhaps when I renamed the branch?